02/app.js

```diff
@@ -1,2 +1,17 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+for (i = 0; i <= x; i++) {
+  result = result + i;
+  console.log(result);
+}
+console.log("Suma wynosi " + result);
+
+let result2 = 0;
+j = 0;
```

Comment:
Tutaj również `let` się przyda (co prawda w tym przypadku będzie to zmienna globalna, ale to dobry nawyk) - zawsze przed pierwszym użycie zmiennych trzeba zdefiniować `let/const`